test/distribution/test_distribution_output_serde.py

```diff
@@ -53,6 +55,9 @@
         DeterministicOutput(value=42.0),
         DirichletMultinomialOutput(dim=3, n_trials=5),
         DirichletOutput(dim=4),
+        EmpiricalDistribution(
```

Comment:
Wait: how does this object fit this test? All others are DistributionOutput

Reply:
updated it..